Ginger/GingerCore/ALM/ZephyrEntCore.cs

```diff
@@ -0,0 +1,338 @@
+﻿using ALM_Common.Data_Contracts;
+using ALM_Common.DataContracts;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Repository;
+using GingerCore.Activities;
+using GingerCore.ALM.QC;
+using GingerCore.ALM.QCRestAPI;
+using GingerCore.ALM.ZephyrEnt.Bll;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OctaneSDK.Services.Queries;
+using QCRestClient;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ZephyrEntSDK.Models;
+using ZephyrEntSDK.Models.Base;
+
+namespace GingerCore.ALM
+{
+    public class ZephyrEntCore : ALMCore
+    {
+        protected Zepyhr_Ent_Repository.ZephyrEntRepository zephyrEntRepository;
+        private ZephyrEntExportManager zephyrEntExportManager;
+        private ZephyrEntImportManager zephyrEntImportManager;
+        public override ObservableList<ActivitiesGroup> GingerActivitiesGroupsRepo { get; set; }
+        public override ObservableList<Activity> GingerActivitiesRepo { get; set; }
+        public ZephyrEntCore()
+        {
+            string settingsPath = DefaultAlmConfig.ALMConfigPackageFolderPath;
```

Comment:
settingspath is unused here. Please remove if not required

Reply:
done